implementation/src/main/java/io/smallrye/metrics/setup/config/HistogramBucketMaxConfiguration.java

```diff
@@ -0,0 +1,62 @@
+package io.smallrye.metrics.setup.config;
+
+import java.util.ArrayDeque;
+import java.util.Collection;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class HistogramBucketMaxConfiguration extends PropertySingleValueConfiguration<Double> {
+
+    private static final String CLASS_NAME = HistogramBucketConfiguration.class.getName();
```

Comment:
Should be `private static final String CLASS_NAME = HistogramBucketMaxConfiguration.class.getName();`